visits/models.py

```diff
@@ -0,0 +1,11 @@
+from django.db import models
+from plates.models import Plate
+from providers.models import Provider
+from visitors.models import Visitor
+
+
+class Visit(models.Model):
+    date = models.DateField()
+    provider = models.ForeignKey(Provider, on_delete=models.CASCADE, null=True)
+    visitor = models.ForeignKey(Visitor, on_delete=models.CASCADE, null=True)
+    plate = models.ForeignKey(Plate, on_delete=models.CASCADE, null=True)
```

Comment:
```suggestion
    provider = models.ForeignKey(Provider, on_delete=models.SET_NULL, null=True)
    visitor = models.ForeignKey(Visitor, on_delete=models.SET_NULL, null=True)
    plate = models.ForeignKey(Plate, on_delete=models.SET_NULL, null=True)
```

So we don't lose the logs of visits once a user no longer exists